connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -494,7 +492,8 @@ private void expectTopicCreation(String topic) {
         }
     }
 
-    private void createSinkTask(TargetState initialState, RetryWithToleranceOperator retryWithToleranceOperator) {
+    private void createSinkTask(TargetState initialState, RetryWithToleranceOperator retryWithToleranceOperator,
+                                Supplier<List<ErrorReporter>> errorReportersSupplier) {
```

Comment:
Nit: do we have to accept a `Supplier` here or can we change the last parameter to `List<ErrorReporter> errorReporters` and pass `() -> errorReporters` to the `WorkerSinkTask` constructor below?